### ISSUE TITLE
ofxAccelerometer has unimplemented functions.

### ISSUE THREAD

**Issue by bakercp** [openframeworks/openFrameworks#5443]
https://github.com/openframeworks/openFrameworks/blob/master/addons/ofxAccelerometer/src/ofxAccelerometer.h#L48-L52

Causes some older examples to faile to compile.

**Comment by bakercp** [comment]
In the master branch, that is.

**Comment by arturoc** [comment]
it's kind of strange and probably we need to change how it works but those methods should be implemented in android and iOS through:

https://github.com/openframeworks/openFrameworks/blob/master/addons/ofxAccelerometer/src/ofxAccelerometer.h#L186

**Comment by bakercp** [comment]
Yes, I assumed that was likely the case. Nevertheless, when generating a new iOS project with master branch and attempting to compile, code like

https://github.com/vanderlin/ofxBox2d/blob/stable/example-iOS/src/testApp.mm#L7

It failed with unimplemented linker errors.  My solution was just to use the core motion.

https://github.com/vanderlin/ofxBox2d/blob/master/example-iOS/src/ofApp.mm#L6-L9

But anyway, not sure then if this is a project file construction error, or something else.
